docs/Guide/plugins/i18next/getting-started.mdx

```diff
@@ -142,7 +142,7 @@ export class PingCommand extends Command {
   }
 
   public async messageRun(message: Message) {
-    await message.channel.send(await resolveKey('ping:success'));
+    await message.channel.send(await resolveKey(message.guild!, 'ping:success'));
```

Comment:
You can use message:
```suggestion
    await message.channel.send(await resolveKey(message, 'ping:success'));
```

- https://github.com/sapphiredev/plugins/blob/5c855bd8341f155a41c9b85738541f1f47ac837a/packages/i18next/src/lib/functions.ts#L57-L63
- https://github.com/sapphiredev/plugins/blob/5c855bd8341f155a41c9b85738541f1f47ac837a/packages/i18next/src/lib/functions.ts#L19-L23